src/App.js

```diff
@@ -132,7 +132,9 @@ const WrappedApp = () => {
   if (client)
     return (
       <CozyProvider client={client}>
-        <Nav client={client} setClient={setClient} />
+        <SharingProvider>
```

Comment:
No way we call that SharingProvider, like I said you yesterday. We already have a `SharingProvider` https://github.com/cozy/cozy-libs/blob/master/packages/cozy-sharing/src/SharingProvider.jsx that doesn't do the same thing.